.circleci/configurations/jobs.yml

```diff
@@ -129,12 +129,8 @@ jobs:
       - run:
           name: Install appium drivers
           command: |
-            appium driver install uiautomator2
-            appium driver install xcuitest
-      - run:
-          name: Start Appium server
-          command: appium --base-path /wd/hub
-          background: true
+            appium driver install uiautomator2@2.29.0
+            appium driver install xcuitest@5.6.0
```

Comment:
Let's update docs, so that when somebody tries to setup e2e tests locally will have same versions. 

https://github.com/facebook/react-native/blob/3648886ff03a2e2aa8bc124a8b1e76c595e4db18/packages/rn-tester-e2e/README.md?plain=1#L22-L23